web/adelacecilia/html/footer/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <title>Footer</title>
+<body>
+
+<h1>The "footer" tag</h1>
```

Comment:
Please dont leave empty lines in your HTML :)

Reply:
please delete blank line 5